source/slang/hlsl.meta.slang

```diff
@@ -2568,12 +2568,24 @@ __target_intrinsic(cuda, "__ballot_sync($0, $1)")
 __target_intrinsic(hlsl, "WaveActiveBallot($1)")
 WaveMask WaveMaskBallot(WaveMask mask, bool condition);
 
+// Predeclare this function
+uint _WaveCountBits(uint4 value);
+
+// Ideally we'll do this with WaveMask backing type changing in the future,
+// but for the moment it's stuck on uint. So this is a work around, such that we can make
+// the code Slang code.
 __glsl_extension(GL_KHR_shader_subgroup_ballot)
 __spirv_version(1.3)
-__target_intrinsic(glsl, "bitCount(subgroupBallot($1))")
+__target_intrinsic(glsl, "subgroupBallot($0)")
+uint4 _WaveActiveBallot(bool condition);
```

Comment:
Why can't this just be/use the existing `WaveActiveBallot`? Is there a reason why an underscore-prefixed version is needed?